viewer/packages/camera-manager/src/CameraManager.ts

```diff
@@ -0,0 +1,584 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import * as THREE from 'three';
+import TWEEN from '@tweenjs/tween.js';
+import { ComboControls } from './ComboControls';
+import { CallbackData, CameraControlsOptions, CameraChangeData, PointerEventDelegate, ControlsState } from './types';
+import { assertNever, EventTrigger, InputHandler } from '@reveal/utilities';
+
+export class CameraManager {
```

Comment:
Not sure what to do with that. Can extract some methods with event listeners but it will require passing domElement as an argument, so not sure whether it will look better. As for creating some other classes, we could add Animations class but probably in other PR.

Reply:
There's a few possible low hanging fruit her:
- NearFarPlaneHelper
- CameraAnimationHelper

Since this is moved from viewer its ok if we do this in a separate PR to me, but it would make things nicer.